gnoboard/src/screens/certify/create-password/index.tsx

```diff
@@ -38,36 +41,55 @@ const CreatePassword: React.FC<Props> = ({ route }) => {
     }
 
     try {
+      if (!override) {
+        const existingAccountName = await gno.isAccountNameExists(name);
```

Comment:
If you rebase to the latest main branch, you can use gno.hasKeyByName.
https://github.com/gnolang/gnomobile/blob/9566241f50aea08c0611d410c5aeeb27ba3821a6/gnoboard/src/hooks/use-gno.ts#L93